.github/workflows/windows-nodejs.yml

```diff
@@ -0,0 +1,41 @@
+name: Windows Node CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    env:
+      CI: true
+      CONFIRM_DROP: 1
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14'
+      - name: Start PostgreSQL
+        run: |
+          $postgreSqlSvc = Get-Service "postgresql*"
+          Set-Service $postgreSqlSvc.Name -StartupType manual
+          $postgreSqlSvc.Start()
+          $env:PATH += ";$env:PGBIN"
+          $env:PGPASSWORD = "root"
+          $postgreSqlSvc.WaitForStatus('Running')
+          psql -U postgres -d postgres -c "alter user postgres with password 'postgres'";
+          Restart-Service $postgreSqlSvc
```

Comment:
Is there a way to pin this to a specific PostgreSQL version? I'm concerned that jumping up major versions without explicitly stating to do so may make maintainance of the repo more challenging.

Reply:
That is a fair comment! I will update it with a specific version that can be downloaded from the Windows Package Manager.